test/Microsoft.AspNet.Mvc.IntegrationTests/TestMvcOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    public class TestMvcOptions : IOptions<MvcOptions>
```

Comment:
This already exists elsewhere in code - suggest moving the existing one to TestCommon and using it.